post-processing/plot_handler.py

```diff
@@ -10,6 +10,77 @@
 from bokeh.plotting import figure, output_file, save
 from bokeh.transform import factor_cmap
 from titlecase import titlecase
+import itertools
+
+def plot_line_chart(title, df: pd.DataFrame, x_axis, y_axis, series_filters):
+    """
+        Create a line chart for the supplied data using bokeh.
+
+        Args:
+            title: str, plot title.
+            df: dataframe, data to plot.
+            x_axis: dict, x-axis column and units.
+            y_axis: dict, y-axis column and units.
+            series_filters: list, x-axis groups used to filter graph data.
+    """
+    # get column names and labels for axes
+    x_column, x_label = get_axis_labels(df, x_axis, series_filters)
+    y_column, y_label = get_axis_labels(df, y_axis, series_filters)
+
+    # adjust axis ranges
+    min_y = (0 if min(df[y_column]) >= 0
+             else math.floor(np.nanmin(df[y_column])*1.2))
+    max_y = (0 if max(df[y_column]) <= 0
+             else math.ceil(np.nanmax(df[y_column])*1.2))
+    min_x = (0 if min(df[x_column]) >= 0
+             else math.floor(np.nanmin(df[x_column])*1.2))
+    max_x = (0 if max(df[x_column]) <= 0
+             else math.ceil(np.nanmax(df[x_column])*1.2))
```

Comment:
Note: This will not work for datetime x-axis values. Not sure if that's out of scope for this PR, but might be worth leaving a `#FIXME` comment somewhere.